### ISSUE TITLE
filtering on 'submission type' (and perhaps others) requires 'Clear current filter' before new value can be used for filtering

### ISSUE THREAD

**Issue by smutniak** [TexasDigitalLibrary/Vireo#1592]
also value entry requires double hit on return key

**Comment by kaladay** [comment]
I would note that the current designed behavior is to append anything added to the currently filtering rather than replacing an existing values.

I did some tests in my instance off of the recent sprint changes (for 4.1.4) and I have observed the following:
- The clicking two enters is because the first enter selects from the dropdown rather than applying the filter. This allows one to make changes or not. The second enter commits the action and saves the filter. This seems like by design behavior.
- The filters act as an "Append" rather than a "Replace". Every filter applied adds to the existing. This seems to be by design.
- Existing filters can be removed by selecting them and pressing enter or clicking with the mouse.

See example:

![Screenshots with arrows showing filter flow](https://user-images.githubusercontent.com/35114839/229178183-eacd75f6-c975-4672-8f13-39f21a8292b5.png)
